kong/clustering/version_negotiation/services_requested.lua

```diff
@@ -0,0 +1,6 @@
+return {
+  {
+    name = "Config",
```

Comment:
nit: should we lowercase this and make it case insensitive?

Reply:
done